pkg/util/constants.go

```diff
@@ -28,3 +28,5 @@ const (
 func GetAlternateDNS(domain string) []string {
 	return []string{"kubernetes.default.svc." + domain, "kubernetes.default.svc", "kubernetes.default", "kubernetes"}
 }
+
+const reportingURL = "https://clouderrorreporting.googleapis.com/v1beta1/projects/aprindle-vm/events:report?key=AIzaSyAicHoDP7bg78e60f60L08MTeL96HPvvSg"
```

Comment:
Maybe add a comment that this API key is ok to expose because all it can do is report errors to stackdriver.